core/equations.gms

```diff
@@ -460,80 +460,88 @@ q_limitGeopot(t,regi,peReComp(enty),rlf)..
 q_costTeCapital(t,regi,teLearn)$(NOT (pm_data(regi,"tech_stat",teLearn) eq 4 AND t.val le 2020)) ..
   vm_costTeCapital(t,regi,teLearn)
   =e=
-*** floor costs defined regionally
-  pm_data(regi,"floorcost",teLearn)
 *** until 2005: using global estimates better matches historic values
-  + ( fm_dataglob("learnMult_wFC",teLearn)
-    * ( ( sum(regi2, vm_capCum(t,regi2,teLearn))
-          + pm_capCumForeign(t,regi,teLearn)
-        )
-        ** fm_dataglob("learnExp_wFC",teLearn)
+  + ( fm_dataglob("floorcost",teLearn)
```

Comment:
I remember Fabrice changed the formulation of the inco0 some time ago (before it was inco0-learned cost = floorcost, now it is based on floorcost)
So was this term missing before(?) in the equation?
is it a bug-fix (so the equation is aligned with Fabrice's change)? thanks!